src/roboat_ros_node.py

```diff
@@ -1,12 +1,11 @@
-#!/usr/bin/env python
+#!/home/jitske/Documents/roboat_ws/venv/bin/python3.6
```

Comment:
What is the reason to add this? Python should be in a common location across different machines right?